### ISSUE TITLE
Cant Understand the meaning of content in Improve Detection Accuracy README

### ISSUE THREAD

**Issue by oggyfaker** [AlexeyAB/darknet#7083]
Thank you for your perfect source code. I saw your mention in READ ME like this : 
"""
General rule - your training dataset should include such a set of relative sizes of objects that you want to detect:

train_network_width * train_obj_width / train_image_width ~= detection_network_width * detection_obj_width / detection_image_width
train_network_height * train_obj_height / train_image_height ~= detection_network_height * detection_obj_height / detection_image_height
"""

I don't know what is your meaning about this way ? Is the way of changing accordination in annotation txt file ? Can you explain it and give some example to do it . Thank you too much

**Comment by stephanecharette** [comment]
The size of the objects in the images used for training must be similar to the size of the objects in the images used for detection.

https://www.ccoderun.ca/programming/darknet_faq/#crop_training_images

But this sentence of yours I don't understand:

> Is the way of changing accordination in annotation txt file ?

**Comment by oggyfaker** [comment]
> The size of the objects in the images used for training must be similar to the size of the objects in the images used for detection.
> 
> https://www.ccoderun.ca/programming/darknet_faq/#crop_training_images
> 
> But this sentence of yours I don't understand:
> 
> > Is the way of changing accordination in annotation txt file ?

I confused the formulation of the author  . But after reading yours comment , i know the meaning . But after following the instruction , i don't know  how to improve the model for training . Although i calculate the anchors and mask for my datasets , the loss of model just only stable from 11-10 after 333 interations. I checked my dataset with [-show_imgs] and see some augmetation like this 

![image](https://user-images.githubusercontent.com/44501897/101276385-cb6b3400-37de-11eb-8fe3-73676b1d1ffe.png)



. So can you give me help me to avoid this issue ? and another way for improving accuracy of model ?   

This is my graph

![image](https://user-images.githubusercontent.com/44501897/101276295-716a6e80-37de-11eb-9ad7-f55976280dca.png)

**Comment by stephanecharette** [comment]
> So can you help me to avoid this issue?

...avoid *what* issue exactly?  If you're trying to turn off mosaic, add this line in the `[net]` section of your configuration file:  `mosaic=0`.

**Comment by stephanecharette** [comment]
> [...] the loss of model just only stable from 11-10 after 333 interations

Some question:

- How many images do you have?
- How many classes?
- Which network are you using (e.g., yolov4-tiny?)
- What does chart.png look like after you finished your 6000 iterations?

Looking at the chart after only 352 iterations is useless.  You have to wait until you have several thousand iterations finished.
